odoo/http.py

```diff
@@ -82,16 +82,16 @@ def replace_request_password(args):
         args[2] = '*'
     return tuple(args)
 
+
 # don't trigger debugger for those exceptions, they carry user-facing warnings
 # and indications, they're not necessarily indicative of anything being
 # *broken*
-NO_POSTMORTEM = (odoo.osv.orm.except_orm,
-                 odoo.exceptions.AccessError,
-                 odoo.exceptions.ValidationError,
-                 odoo.exceptions.MissingError,
+NO_POSTMORTEM = (odoo.exceptions.except_orm,
```

Comment:
>  http.py WebRequest handle_exception was not handling HTTPExceptions
>   as we want,

Can we know what you want ;) ?

Commit msg don't explain why you remove it... exactly. And which case/bug/error you handle.

Reply:
This should not have been committed. I was experiencing issues and taught odoo was not handling errors correctly.